db/migrate/20210707151757_populate_lp_and_cpd_lp_assocs.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class PopulateLpAndCpdLpAssocs < ActiveRecord::Migration[6.1]
+  def up
+    LeadProvider.all.each do |lp|
+      lp.update(cpd_lead_provider: CpdLeadProvider.find_by(name: lp.name))
+    end
+
+    NpqLeadProvider.all.each do |lp|
+      lp.update(cpd_lead_provider: CpdLeadProvider.find_by(name: lp.name))
+    end
```

Comment:
Can we do `update!` rather than `update`? I know, I see no way in hell this can fail, but I think it's a good idea to enforce this as a convention.